patternlab/docs/a11y-checklist/headings.md

```diff
@@ -0,0 +1,94 @@
+[⬅️ a11y Checklist](a11y-checklist.md)
+
+# Headings `<h#>`
+
+## Heading level
+
+A heading needs its **proper level number** to represent the hierarchy of the content in the relationship of the parent and child components.
+
+HTML5 allows you to start with `<h1>` in every `<section>`. That is valid as HTML5, but it doesn't help assistive technology users to grasp the content hierarchy in the page.
```

Comment:
We reserves `<h1>` for a page title, which is only used once in a page. The rest of headings in the page should be defined in the relation of the page title `<h1>` because they are supposed to fall under the page thus they are `<h2>`, `<h3>` and so on although some utility type components might not make sense to this rule.  There might be multiple components in the same level in a page, so the page can have multiple `<h2>`s, `<h3>`s and so forth.

The headings to present the content structure of the  page should be defined using the only `<h1>` as their structural starting point.